amgut/handlers/auth_handlers.py

```diff
@@ -23,6 +23,8 @@ def get(self):
     @authenticated
     def post(self):
         skid = self.current_user
+        # log user out for register process
+        self.clear_cookie("skid")
```

Comment:
It does disable the button as well. The logout is for in case it errors, so we don't get that weird limbo state of logged in but no registered kits.